src/Model/TransactionalManagerInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Sonata\Doctrine\Model;
+
+interface TransactionalManagerInterface
+{
+    public function beginTransaction(): void;
+
+    public function commit(): void;
```

Comment:
A commit can fail.

IMHO, we should decide if we want the commit to return true/false for success/failure,
or if we throws an exception for failure.

If we throw an exception, it might be better to add 
```
/** @throws aCustomException */
```
rather than `\Throwable`.

So in the implementation it should be
```
try {
     $this->getEntityManager()->commit();
} catch (Throwable $e) {
     throw new CustomException(...);
}
```

What do you prefer @jordisala1991 ?

Reply:
TransactionException maybe ?

Reply:
Yeah I'm fine with that too 😄